src/qofonomanager.cpp

```diff
@@ -106,6 +106,7 @@ void QOfonoManager::onModemRemoved(const QDBusObjectPath& path)
 void QOfonoManager::onGetModemsFinished(QDBusPendingCallWatcher* watcher)
 {
     QDBusPendingReply<ObjectPathPropertiesList> reply(*watcher);
+    watcher->deleteLater();
```

Comment:
Unfortunately I lost the stack trace but basically, something was handling the `availableChanged()` signal (`QOfonoSimManager` was involved) which resulted in `QOfonoManager` being deleted, `QOfonoManager` is the parent of this particular `QDBusPendingCallWatcher`, so it gets deleted too and by the time we were getting to `watcher->deleteLater()` at the end of the function, `watcher` was already dead.

Moving `deleteLater()` to the top of the function fixes the crash and everything works even if by the time `availableChanged()` returns, `QOfonoManager` is already dead (`this` pointer is not accessed after that call). But it's easy to imagine that other signals (`modemsChanged`, `defaultModemChanged`) delete this `QOfonoManager`, in that case moving `deleteLater()` around wouldn't help.